test/steps/ffmpeg.go

```diff
@@ -21,35 +24,45 @@ func (s *StepContext) CheckFfmpeg() error {
 }
 
 func (s *StepContext) AllOfTheSourceSegmentsAreWrittenToStorageWithinSeconds(numSegments, secs int) error {
-	// Comes in looking like file:/var/folders/qr/sr8gs8916zd2wjbx50d3c3yc0000gn/T/livepeer/source
-	// and we want /var/folders/qr/sr8gs8916zd2wjbx50d3c3yc0000gn/T/livepeer/source/aceaegdf/source/*.ts
-	segmentingDir := filepath.Join(strings.TrimPrefix(s.SourceOutputDir, "file:"), s.latestRequestID, "source/*.ts")
+	osDriver, err := drivers.ParseOSURL(s.SourceOutputDir, true)
+	if err != nil {
+		return fmt.Errorf("could not parse object store url: %w", err)
+	}
+	session := osDriver.NewSession(filepath.Join(s.latestRequestID, "source"))
 
 	var latestNumSegments int
 	for x := 0; x < secs; x++ {
-		files, err := filepath.Glob(segmentingDir)
+		page, err := session.ListFiles(context.Background(), "", "")
 		if err != nil {
-			return err
+			log.Println("failed to list files: ", err)
+			time.Sleep(time.Second)
```

Comment:
This is just for the retry loop, so just a pause before the next retry, as the loop was already doing. I've moved the sleep to the top of the loop so that it's just in one place and we don't have to remember to add it in further places like this.